### ISSUE TITLE
Hunting Girl has no drops

### ISSUE THREAD

**Issue by Fulmir** [Lakifume/SotnKindAndFair#9]
Running only this Rom Hack, running normal settings, no scavenger only, tested with 999 luck and everything else was dropping items. There may be some other enemies that aren't dropping stuff as well, but this was the only one I tested specifically and thoroughly. It's also not dropping hearts or money.

Edit: Seems to also apply to the Vandal Swords that spawn out of the Sword Lords. The Swordies themselves have their drops, but the resulting phantom does not. Again tested on multiple enemies using 999 luck. This causes everything *except* for the bugged enemies to drop some kind of loot every kill.

**Comment by Lakifume** [comment]
Yeah the explanation behind this is that the byte for stopwatch tolerance and loot drops is one and the same, and I have never been able to figure out the relation between these 2 things within that byte. So I just assigned the set values that I knew for stopwatch tolerance which also caused some loot drops to be inaccurate.

**Comment by Lakifume** [comment]
The latest release should now fix this issue
